### PR TITLE
Wba 101 scripturebody and questionbody

### PR DESCRIPTION
**Pull request by YiJie-Zhu**

### CODE REVIEW

File: lib/presentation/engage/bible_series/widgets/scripture_body.dart [hunk 1]
```diff
@@ -10,6 +12,81 @@ class ScriptureContentBodyWidget extends StatelessWidget {
...
+            padding: const EdgeInsets.only(bottom: 8.0),
+            child: verse(script, key),
+          );
+        },
+      ),
+    ],
+  );
+}
+
+Widget title(Scripture script) {
```
Comment (JoshuaRamsamooj) [inline lib/presentation/engage/bible_series/widgets/scripture_body.dart]: There is also an optional `title` parameter in the Scripture class that should be used here. Sorry if it's not included in the Figma. You can put it just below the verse information.

Comment (JoshuaRamsamooj) [reply lib/presentation/engage/bible_series/widgets/scripture_body.dart]: I updated the Figma to reflect this.

Comment (YiJie-Zhu) [reply lib/presentation/engage/bible_series/widgets/scripture_body.dart]: ok got it

File: lib/presentation/engage/bible_series/widgets/scripture_body.dart [hunk 2]
```diff
@@ -10,6 +12,89 @@ class ScriptureContentBodyWidget extends StatelessWidget {
...
 }
+
+Widget scripture(Scripture script) {
+  return Column(children: buildcombinedScripture(script));
+}
+
+List<Widget> buildcombinedScripture(Scripture script) {
+  List<Widget> combinedScripture = [];
+  combinedScripture.add(book(script));
+  if (script.title != null) {
```
Comment (JoshuaRamsamooj) [inline lib/presentation/engage/bible_series/widgets/scripture_body.dart]: In infrastructure/bible_series/series_content_dtos.dart#L180 the `title` actually defaults to an empty string so we can either make that default to null or check `if != ''`

**Review by JoshuaRamsamooj — APPROVED** [review]

**Pull request merged**